# This file is distributed under the same license as the Django package.
#
# Translators:
# crazyzubr <hjcnbckfd@gmail.com>, 2020
# Ivan Khomutov <iskhomutov@gmail.com>, 2017
# Jannis Leidel <jannis@leidel.info>, 2011
# Алексей Борискин <sun.void@gmail.com>, 2012-2015
# Андрей Щуров <shurov9009@gmail.com>, 2016
# Дмитрий Шатера <mr.bobsans@gmail.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-05-24 10:21+0500\n"
"PO-Revision-Date: 2020-05-13 18:48+0000\n"
"Last-Translator: crazyzubr <hjcnbckfd@gmail.com>\n"
"Language-Team: Russian (http://www.transifex.com/django/django/language/"
"ru/)\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
"%100>=11 && n%100<=14)? 2 : 3);\n"

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr "Персональная информация"

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr "Права доступа"

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:50
msgid "Important dates"
msgstr "Важные даты"

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "%(name)s с первичным ключом %(key)r не существует."

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:144
msgid "Password changed successfully."
msgstr "Пароль успешно изменен."

#: venv/lib/python3.9/site-packages/django/contrib/auth/admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "Изменить пароль: %s"

#: venv/lib/python3.9/site-packages/django/contrib/auth/apps.py:16
msgid "Authentication and Authorization"
msgstr "Пользователи и группы"

#: venv/lib/python3.9/site-packages/django/contrib/auth/base_user.py:49
msgid "password"
msgstr "пароль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/base_user.py:50
msgid "last login"
msgstr "последний вход"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:41
msgid "No password set."
msgstr "Пароль не задан."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:46
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Неизвестный формат пароля или алгоритм хеширования."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:81
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:322
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:395
msgid "The two password fields didn’t match."
msgstr "Введенные пароли не совпадают."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:84
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:137
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:167
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:399
msgid "Password"
msgstr "Пароль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:90
msgid "Password confirmation"
msgstr "Подтверждение пароля"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:93
#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:408
msgid "Enter the same password as before, for verification."
msgstr "Для подтверждения введите, пожалуйста, пароль ещё раз."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:139
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Пароли хранятся в зашифрованном виде, поэтому нет возможности посмотреть "
"пароль этого пользователя, но вы можете изменить его используя <a href="
"\"{}\">эту форму</a>."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:174
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Пожалуйста, введите правильные %(username)s и пароль. Оба поля могут быть "
"чувствительны к регистру."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:177
msgid "This account is inactive."
msgstr "Эта учетная запись отключена."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:240
msgid "Email"
msgstr "Адрес электронной почты"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:325
msgid "New password"
msgstr "Новый пароль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:331
msgid "New password confirmation"
msgstr "Подтверждение нового пароля"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:367
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Ваш старый пароль введен неправильно. Пожалуйста, введите его снова."

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:370
msgid "Old password"
msgstr "Старый пароль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/forms.py:405
msgid "Password (again)"
msgstr "Пароль (еще раз)"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:297
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:388
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:477
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:544
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:586
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:633
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:677
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:723
msgid "algorithm"
msgstr "алгоритм"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:298
msgid "iterations"
msgstr "итерации"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:299
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:394
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:479
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:545
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:587
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:724
msgid "salt"
msgstr "соль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:300
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:395
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:546
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:588
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:634
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:678
#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:725
msgid "hash"
msgstr "хэш"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:389
msgid "variety"
msgstr "множество"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:390
msgid "version"
msgstr "версия"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:391
msgid "memory cost"
msgstr "затраты памяти"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:392
msgid "time cost"
msgstr "затраты времени"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:393
msgid "parallelism"
msgstr "параллелизм"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:478
msgid "work factor"
msgstr "рабочий фактор"

#: venv/lib/python3.9/site-packages/django/contrib/auth/hashers.py:480
msgid "checksum"
msgstr "контрольная сумма"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:58
#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:109
msgid "name"
msgstr "имя"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:62
msgid "content type"
msgstr "тип содержимого"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:64
msgid "codename"
msgstr "кодовое название"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:69
msgid "permission"
msgstr "право"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:70
#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:112
msgid "permissions"
msgstr "права"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:119
msgid "group"
msgstr "группа"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:120
#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:247
msgid "groups"
msgstr "группы"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:238
msgid "superuser status"
msgstr "статус суперпользователя"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:241
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr "Указывает, что пользователь имеет все права без явного их назначения."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:250
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Группы, к которым принадлежит данный пользователь. Пользователь получит все "
"права, указанные в каждой из его/её групп."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:258
msgid "user permissions"
msgstr "права пользователя"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:260
msgid "Specific permissions for this user."
msgstr "Индивидуальные права данного пользователя."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:331
msgid "username"
msgstr "имя пользователя"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:334
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Обязательное поле. Не более 150 символов. Только буквы, цифры и символы @/./"
"+/-/_."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:337
msgid "A user with that username already exists."
msgstr "Пользователь с таким именем уже существует."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:340
msgid "first name"
msgstr "имя"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:341
msgid "last name"
msgstr "фамилия"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:342
msgid "email address"
msgstr "адрес электронной почты"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:344
msgid "staff status"
msgstr "статус персонала"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:346
msgid "Designates whether the user can log into this admin site."
msgstr ""
"Отметьте, если пользователь может входить в административную часть сайта."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:349
msgid "active"
msgstr "активный"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:352
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Отметьте, если пользователь должен считаться активным. Уберите эту отметку "
"вместо удаления учётной записи."

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:356
msgid "date joined"
msgstr "дата регистрации"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:365
msgid "user"
msgstr "пользователь"

#: venv/lib/python3.9/site-packages/django/contrib/auth/models.py:366
msgid "users"
msgstr "пользователи"

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символ."
msgstr[1] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символа."
msgstr[2] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."
msgstr[3] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "Ваш пароль должен содержать как минимум %(min_length)d символ."
msgstr[1] "Ваш пароль должен содержать как минимум %(min_length)d символа."
msgstr[2] "Ваш пароль должен содержать как минимум %(min_length)d символов."
msgstr[3] "Ваш пароль должен содержать как минимум %(min_length)d символов."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Введённый пароль слишком похож на %(verbose_name)s."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can’t be too similar to your other personal information."
msgstr "Пароль не должен быть слишком похож на другую вашу личную информацию."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:183
msgid "This password is too common."
msgstr "Введённый пароль слишком широко распространён."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:188
msgid "Your password can’t be a commonly used password."
msgstr "Такой пароль часто используется."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:198
msgid "This password is entirely numeric."
msgstr "Введённый пароль состоит только из цифр."

#: venv/lib/python3.9/site-packages/django/contrib/auth/password_validation.py:203
msgid "Your password can’t be entirely numeric."
msgstr "Пароль не может состоять только из цифр."

#: venv/lib/python3.9/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Сброс пароля на %(site_name)s"

#: venv/lib/python3.9/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Это значение может содержать только "
"английские буквы, цифры и знаки @/./+/-/_."

#: venv/lib/python3.9/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Оно может содержать только буквы, цифры "
"и знаки @/./+/-/_."

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:160
msgid "Logged out"
msgstr "Не авторизован"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:217
msgid "Password reset"
msgstr "Восстановление пароля"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:244
msgid "Password reset sent"
msgstr "Письмо с инструкциями по восстановлению пароля отправлено"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:254
msgid "Enter new password"
msgstr "Введите новый пароль"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:314
msgid "Password reset unsuccessful"
msgstr "Ошибка восстановления пароля"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:322
msgid "Password reset complete"
msgstr "Восстановление пароля завершено"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:334
msgid "Password change"
msgstr "Изменение пароля"

#: venv/lib/python3.9/site-packages/django/contrib/auth/views.py:357
msgid "Password change successful"
msgstr "Пароль успешно изменен"
